README.rst

```diff
@@ -184,7 +184,7 @@ project:
 
     $ php php-cs-fixer.phar fix /path/to/project --rules=@PSR2
 
-By default the PSR1 and PSR2 rules are used.
+If the ``--rules`` option is used, ``.php_cs`` is ignored. By default the PSR1 and PSR2 rules are used.
```

Comment:
Actually, It's not correct either: only rules are ignored and also config file can have different name (`.php_cs.fist).

I'd suggest:

If the ``--rules`` option is used, rules from config files are ignored.